shared/teams/team/members/member-row/index.js

```diff
@@ -45,20 +46,21 @@ export const TeamMemberRow = (props: Props) => {
     fullName,
     onChat,
     onClick,
-    type,
+    roleType,
     username,
```

Comment:
can we just change this to props: Props and not have very long destructuring. i think it makes it less readable and doesn't really serve a purpose